### DISCUSSION TITLE
blog/lz7xgx/

### DISCUSSION THREAD

**Discussion by giscus** [wztlink1013/comment#46] [General]
# blog/lz7xgx/

理解js里面的任务执行机制相关概念异步编程Javascript的事件循环任务队列微任务队列……以下代码执行的结果是什么？console.log(1);setTimeout(function () {console.log(3);}, 3000);console.log(2);以下代码执行的结果是什么

http://localhost:4000/blog/lz7xgx/

**Comment by wztlink1013** [comment]
写一条评论

**Comment by wztlink1013** [comment]
再来一条
